webapp/components/admin_console/brand_image_setting.jsx

```diff
@@ -40,6 +40,19 @@ export default class BrandImageSetting extends React.Component {
         });
     }
 
+    componentDidUpdate() {
+        if (this.refs.image) {
+            const reader = new FileReader();
```

Comment:
Looks like all the browsers we support support this https://developer.mozilla.org/en/docs/Web/API/FileReader so I'm going to skip it for now